charts/stable/onedrive/values.yaml

```
@@ -9,7 +9,7 @@ image:
   # -- image repository
   repository: ghcr.io/wrmilling/onedrive-docker
   # -- image tag
-  tag: v2.0.5
+  tag: v2.4.17
```

```suggestion
  tag:
```

By default it will use the appVersion in the Chart.yaml, this makes it so we only need to manage the version in one place.

---

Was not aware it would do that, neato.